test/utils/image/manifest.go

```diff
@@ -116,3 +122,12 @@ func GetE2EImageWithArch(image ImageConfig, arch string) string {
 func GetPauseImageName() string {
 	return GetE2EImage(Pause)
 }
+
+// GetImageSizes returns a map of an image name to its (virtual) size in bytes. This is used primarily in
+// the integration-test, testing image locality function which requires image size information. The image sizes do not
+// have to match the real sizes in the registry.
+func GetImageSizes() map[string]int64 {
```

Comment:
Only return `MnistTpu`, prefer not to use such a general name.

Reply:
if it is not a real size then why don't you mention that number directly in the testcase instead of defining here?